### ISSUE TITLE
[UI] Designs: Change color of checkboxes on table column list

### ISSUE THREAD

**Issue by leecalcote** [meshery/meshery#6569]
#### Current Behavior
On the Designs page, this purple checkbox color should not be used:

<img width="468" alt="Screenshot 2022-11-24 at 1 48 43 PM" src="https://user-images.githubusercontent.com/7570704/203856042-eb5a7c54-e0b7-4637-8e1d-dc7caf0a92e3.png">

#### Desired Behavior
The color #3C494F should be used instead.

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

**Comment by ashtonicci** [comment]
Can I take a look at this? Thanks :)

**Comment by partik03** [comment]
@leecalcote Can I take up this one??

**Comment by leecalcote** [comment]
Oh, no. I missed @ashtonicci’s message above. Yes, this is still an issue and would be good for either of you to solve @partik03. Alice and both of you given that it is a small item and either one of the two of you can PR it.

**Comment by Swarno-Coder** [comment]
Can I take up this one? @leecalcote

**Comment by leecalcote** [comment]
@Swarno-Coder, yes, let's see.... @partik03, how is this coming along? Help needed? 

@Swarno-Coder, why don't you give it a shot and see if @partik03 is open to collaborate, too. Minimally, one of you can review the other's PR.
